### ISSUE TITLE
Separate Real BluRays From 1080P Rips/Encodes 

### ISSUE THREAD

**Issue by TRaSH-** [Radarr/Radarr#3011]
**Desktop (please complete the following information):**
 - OS: Debian 9
 - Mono Version: 5.14.0.177
 - Browser and Version [e.g. chrome, safari] (Only needed for UI issues)
 - Version: 0.2.0.1120

**Expected behavior**
Radarr should't see 1080p-Rips/Encodes as blurays just because the word bluray is in it.

I've asked the same question on [reddit](https://www.reddit.com/r/radarr/comments/9dm3cj/how_to_separate_real_blurays_from_1080p/)
and they told me to open a ticket here.

Radarr marks everything with the word bluray in the filename as bluray,
and actually it should see 1080p x264 encodes as 1080p-rips/encode.
The word bluray in the encodes mainly means it's a rip/encode of the bluray.

I collect different quality's depending what kind of movie it is.

Mostly i want full blurays for the quality and HD audio.
But for pixar/disney or comedy 1080p rips will do fine as long as they got DTS audio.

First i thought i needed to use the profile BR-Disk but nothing get's marked as BR-Disk.
looking at the following screenshot
![image](https://user-images.githubusercontent.com/6155095/45251583-e2452680-b348-11e8-8bc2-664aad3bdbf8.png)
everything is marked as bluray,
the first 2 are full blurays,
the others are 1080p rips/encodes (notice the x264 meaning it's a rip/encode)
(And yes the first 3 are cetainly full blurays)

So messing with the " Quality Definitions " won't work because when i make the minimum size to big it won't recognize the 1080p rips i sometimes want.
using different profiles will not work because the 1080p rips/encodes are still seen as blurays.

only way to make sure it sees the full blurays as blurays  and 1080p rips/encodes is with a bunch of tags with ignore and must have.
Let me know if you need more examples or anything else.
or you want me to copy/paste the complete reddit discussion/examples here.

**Comment by galli-leo** [comment]
We do mark br disks, but those aren’t ones. If they are, tell your indexer to follow scene naming rules and tag them as such (e.g. with BD50). We don’t have a chance correctly identifying full blurays else. Custom Formats should help you though.

**Comment by TRaSH-** [comment]
I can tell for sure those are full blurays,
MTeam and HDCHina are one are one of the major bluray groups on P2P that also eventually comes to usenet.

I got several indexers in use and did a search on BD50 and it can only find a small collection of releases that use BD50 in it.
I did notice that full blurays have AVC and VC-1 but that's the used video codec (if i'm not mistaking)
So it seems the so called scene isn't much into real full blurays.

So the only way to make Radarr handle Blurays correctly would be to add to every wanted bluray a tag with "Must Not Contain:  x264" and that would solve the issue beeing that x264 aren't blurays but encodes.

Thnx i'll be using Tags then to solve the issue.
